README.md

```diff
@@ -32,6 +32,25 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+The project is set up to integrate with a Trello board. In the `.env`, you will need to enter in your API key and token as well as the board ID. You can find your API key and token on Trello's dedicated [Developer API Keys](https://trello.com/app-key) webpage.
```

Comment:
Minor, but it might be nice to update your `.env.template` to include these variables.

Other than that this is a very comprehensive and helpful README update - great work!